.gitignore

```
@@ -0,0 +1,5 @@
+node_modules
+.vscode
```

maybe it would be better to use a template... but however for now please add my ide to the gitignore as well... thanks!

```suggestion
.vscode
.idea
```